internal/service/apigatewayv2/stage_unit_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package apigatewayv2
```

Comment:
Since I'm verifying implementation details of a component that is not really well decomposed into pure/injectable collaborators, and I do not have access to an integration testing environment, I provided unit tests that verify the bogus code directly.